couchdb-setup.sh

```diff
@@ -92,8 +92,8 @@ curl -X PUT $COUCHURL/_global_changes
 curl -X PUT $COUCHURL/meetups
 curl -X PUT $COUCHURL/resources
 curl -X PUT $COUCHURL/courses
-curl -X PUT $COUCHURL/nations
-curl -X PUT $COUCHURL/communityregistrationrequests
+curl -X PUT $COUCHURL/childPlanets
+curl -X PUT $COUCHURL/registrationRequests
```

Comment:
could we keep the name convention and make it 'snake_case'

Reply:
Ok, let's go with `child_planets` and `registration_requests`